commands/initial/templates/examples/example.component.html

```diff
@@ -0,0 +1 @@
+<div class="welcome">Welcome</div>
```

Comment:
Yeah, we can scaffold out the skeleton of it and let devs add their own code. I think of how the CLI scaffolds out your initial components & HTML...it annoys me that everytime I create an app I have to remove all those files.